text/0000-read-only-build-layers.md

```diff
@@ -59,6 +59,8 @@ Updates to the lifecycle so that `build` layers created by buildpacks should be
 
 The lifecycle will have to layerize the layers marked `build` as `true` during the `build` phase of the lifecycle instead of the `export` phase.
```

Comment:
We only need to layerize if `cache=true` or `launch=true`. I.E we don't actually need to layerize a `build=true`, `cache=false`, `launch=false` layer because we wouldn't do anything with the resulting layer.